src/Map/src/Bridge/Google/assets/dist/map_controller.js

```diff
@@ -1,110 +1,8 @@
-        this.polylinesValue.forEach((definition) => this.createPolyline({ definition }));
-        if (this.fitBoundsToMarkersValue) {
-            this.doFitBoundsToMarkers();
-        }
-        this.dispatchEvent('connect', {
-            map: this.map,
-            markers: [...this.markers.values()],
-            polygons: [...this.polygons.values()],
-            polylines: [...this.polylines.values()],
-            infoWindows: this.infoWindows,
-        });
-        this.isConnected = true;
-    }
-    createInfoWindow({ definition, element, }) {
-        this.dispatchEvent('info-window:before-create', { definition, element });
-        const infoWindow = this.doCreateInfoWindow({ definition, element });
-        this.dispatchEvent('info-window:after-create', { infoWindow, element });
-        this.infoWindows.push(infoWindow);
-        return infoWindow;
-    }
-    markersValueChanged() {
-        if (!this.isConnected) {
-            return;
-        }
-        this.onDrawChanged(this.markers, this.markersValue, this.createMarker, this.doRemoveMarker);
-        if (this.fitBoundsToMarkersValue) {
-            this.doFitBoundsToMarkers();
-        }
-    }
-    polygonsValueChanged() {
-        if (!this.isConnected) {
-            return;
-        }
-        this.onDrawChanged(this.polygons, this.polygonsValue, this.createPolygon, this.doRemovePolygon);
-    }
-    polylinesValueChanged() {
-        if (!this.isConnected) {
-            return;
-        }
-        this.onDrawChanged(this.polylines, this.polylinesValue, this.createPolyline, this.doRemovePolyline);
-    }
-    createDrawingFactory(type, draws, factory) {
-        const eventBefore = `${type}:before-create`;
-        const eventAfter = `${type}:after-create`;
-        return ({ definition }) => {
-            this.dispatchEvent(eventBefore, { definition });
-            const drawing = factory({ definition });
-            this.dispatchEvent(eventAfter, { [type]: drawing });
-            draws.set(definition['@id'], drawing);
-            return drawing;
-        };
-    }
-    onDrawChanged(draws, newDrawDefinitions, factory, remover) {
-        const idsToRemove = new Set(draws.keys());
-        newDrawDefinitions.forEach((definition) => {
-            idsToRemove.delete(definition['@id']);
-        });
-        idsToRemove.forEach((id) => {
-            const draw = draws.get(id);
-            remover(draw);
-            draws.delete(id);
-        });
-        newDrawDefinitions.forEach((definition) => {
-            if (!draws.has(definition['@id'])) {
-                factory({ definition });
-            }
-        });
-    }
-}
-default_1.values = {
-    providerOptions: Object,
-    center: Object,
-    zoom: Number,
-    fitBoundsToMarkers: Boolean,
-    markers: Array,
-    polygons: Array,
-    polylines: Array,
-    options: Object,
-};
+import AbstractMapController from '@symfony/ux-map';
```

Comment:
I don't know what happens here, but can you ensure that you use Node.js 20 or 22, ensure your JS dependencies are up-to-date, and re-run `yarn build`?

The `@symfony/ux-map` package should be inlined